config/initializers/friendly_id.rb

```diff
@@ -0,0 +1,88 @@
+# FriendlyId Global Configuration
```

Comment:
It autogenerated this file - shall I leave it as is, or delete the commented cruft?